src/arch/riscv/inc/arch/interrupts.h

```diff
@@ -12,6 +12,9 @@
 #define PLIC  (1)
 #define APLIC (2)
 
+#define IPIC_SBI (1)
+#define IPIC_ACLINT (2)
```

Comment:
I think the selection of IPIC should be circumscribed to the qemu platforms files (see other comment)